CONTRIBUTING.md

```diff
@@ -116,14 +116,24 @@ dtls_get_peer(const dtls_context_t *ctx, const session_t *session) {
 ...
 }
 ```
+* If the function definition exceeds the line length of 80 characters caused by
+  the parameter-list, the split this list on multiple lines aligning with the 
+  first parameter.
 
+```
+int
+dtls_read(const dtls_context_t *ctx, const session_t *session,
+          uint8_t* buffer, size_t buffer_length) {
+...
+}
+```
 * Declarations in header files do not follow the previous rule. For
   example, the declaration for `dtls_get_peer()` in `dtls.h` reads as
   follows:
 
 ```
 dtls_peer_t *dtls_get_peer(const dtls_context_t *context,
-			   const session_t *session);
+             const session_t *session);
```

Comment:
This is not right.  Previous TABS have been converted to double spaces, messing up the indentation of the second line.

Reply:
I would have expected the first character of the second line (and subsequent lines if appropriate) to be aligned with the first parameter of the parameter list in the same way as it is done in the *.c files.

One of the things we have been looking at is [clang-format](https://clang.llvm.org/docs/ClangFormat.html) to be consistent with styles.  clang-format is pretty flexible in its style configurations, so lots of room for negotiation there!